src/NodeHandler.ts

```diff
@@ -0,0 +1,44 @@
+import { Evented } from '@dojo/core/Evented';
+import { VNodeProperties } from '@dojo/interfaces/vdom';
+import Map from '@dojo/shim/Map';
+
+export enum Type {
+	Projector = 'Projector',
+	Widget = 'Widget'
+}
+
+export default class NodeHandler extends Evented {
```

Comment:
Can we make this a named export and export default below (a pattern for most widget-core modules)